flycheck.el

```diff
@@ -6972,6 +6973,59 @@ See URL `http://stylelint.io/'."
   :error-parser flycheck-parse-stylelint
   :modes (css-mode))
 
+(flycheck-def-option-var flycheck-cuda-language-standard nil cuda-nvcc
+  "Our CUDA Language Standard "
```

Comment:
Is there a reason the docstring ends with a space and not with a dot `.`?  If not, please use a dot instead, as the first line of a docstring should be a complete sentence.

The same applies to the `flycheck-cuda-includes` option below.

```suggestion
  "Our CUDA Language Standard."
```